docs/upgrades.md

```diff
@@ -74,6 +74,7 @@ required by the latest version (See the section
 ["From vX1.X2.X3 to vY1.Y2.Y3"](#from-vx1x2x3-to-vy1y2y3) below). Then:
 
 ```bash
+helm repo update
 helm upgrade sematic-server -f /path/to.values.yaml
```

Comment:
```suggestion
$ helm repo update
$ helm upgrade sematic-server -f /path/to.values.yaml
```

Reply:
I've found that including those `$ ` can actually be annoying because there is a "copy paste" button in the rendered docs that copies the whole text box. If there are `$ `, then what you copy can't be pasted in the terminal